staging/src/k8s.io/client-go/dynamic/dynamicinformer/interface.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dynamicinformer
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic/dynamiclister"
+	"k8s.io/client-go/tools/cache"
+)
+
+type DynamicInformer interface {
+	Informer() cache.SharedIndexInformer
+	Lister()  dynamiclister.Lister
+}
+
+type DynamicSharedInformerFactory interface {
+	// TODO:
+	//Start(stopCh <-chan struct{})
```

Comment:
If yes should we also provide `Start` and `WaitForCacheSync` methods ?

Reply:
> If yes should we also provide `Start` and `WaitForCacheSync` methods ?

Yes.